### ISSUE TITLE
mel channels

### ISSUE THREAD

**Issue by Crabbit-F** [mindslab-ai/univnet#5]
hello，how can i change mel channels from 80 to 100 to use your model？

**Comment by wookladin** [comment]
Hello, sorry for the late reply.

If you want to use Univnet with your own acoustic model, make sure your acoustic model uses the same mel configuration as Univnet.

For example, let's say if you want to use [Tacotron2](https://github.com/NVIDIA/tacotron2).
Then you should modify mel channels in [this line](https://github.com/NVIDIA/tacotron2/blob/master/hparams.py#L40) from 80 to 100.
Then you have to train Tacotron2 with these configs.

Among acoustic models and vocoders, we recommend you to match the configs of the model, which takes less time to learn, to the pre-trained weights of other models.

**Comment by Crabbit-F** [comment]
thanks a lot

**Comment by Selimonder** [comment]
Alternatively, you can update input convolutions (init from scratch) and fine-tune the model to adjust your resolution. This should take less time for those who have limited computing.

`        # modifying kernels to increase mel resolution
        print("modifying kernels to increase mel resolution")
        cond_channels = 104
        kpnet_hidden_channels = 64
        model_g.res_stack[0].kernel_predictor.input_conv = nn.Sequential(
            nn.utils.weight_norm(nn.Conv1d(cond_channels, kpnet_hidden_channels, 5, padding=2, bias=True)),
            nn.LeakyReLU(0.1),
        )
    
        model_g.res_stack[1].kernel_predictor.input_conv = nn.Sequential(
            nn.utils.weight_norm(nn.Conv1d(cond_channels, kpnet_hidden_channels, 5, padding=2, bias=True)),
            nn.LeakyReLU(0.1),
        )
        model_g.res_stack[1].kernel_predictor.input_conv = nn.Sequential(
            nn.utils.weight_norm(nn.Conv1d(cond_channels, kpnet_hidden_channels, 5, padding=2, bias=True)),
            nn.LeakyReLU(0.1),
        )
        model_g.res_stack[2].kernel_predictor.input_conv = nn.Sequential(
            nn.utils.weight_norm(nn.Conv1d(cond_channels, kpnet_hidden_channels, 5, padding=2, bias=True)),
            nn.LeakyReLU(0.1),
        )`
